crates/runtime/Cargo.toml

```diff
@@ -98,6 +98,7 @@ prost = { version = "0.13.1", default-features = false, features = [
     "prost-derive",
 ] }
 rand = "0.8.5"
+ratelimit = "0.10.0"
```

Comment:
Have you considered https://docs.rs/governor/0.8.0/governor/ instead?

Reply:
Thank you @phillipleblanc - despite both crates are very popular I found `governon` crate is a better choice so updated PR to switch from `ratelimit` to `governon`. https://github.com/spiceai/spiceai/pull/4117/commits/1b100cb239674674482efbfe0cb9cf4d06dc067b